opendata_module/opmon_opendata/api/postgresql_manager.py

```diff
@@ -33,9 +33,13 @@ def __init__(self, settings):
         self._connection_string = self._get_connection_string()
         self._field_name_map = self._get_field_name_map(settings['opendata']['field-descriptions'].keys())
         self._logs_time_buffer = relativedelta.relativedelta(days=settings['opendata']['delay-days'])
+        self._connect_args = {
+            'sslmode': settings['postgres'].get('ssl-mode'),
+            'sslrootcert': settings['postgres'].get('ssl-root-cert')
```

Comment:
Won't there be a problem in case one parameter is provided, but the other is None?

Reply:
`sslrootcert` provided alone will be ignored as my local tests showed.
`sslmode` will require `sslrootcert` set only if `sslmode` values are `verify-ca` or `verify-full` and root certificate is not found in default location.